app/config/parameters.yml.dist

```diff
@@ -11,11 +11,11 @@ parameters:
     coffee.node: ''
 
     database_driver: pdo_sqlite
-    database_host:
-    database_port:
-    database_name:
-    database_user:
-    database_password:
+    database_host: localhost
+    database_port: 3306
```

Comment:
I think we should keep that `.dist`as it is and use a separate one for the box with complete `pdo_mysql` as driver for the mysq created in the box. The `pdo_sqlite` is used for an easy entry point.